src/ansys/fluent/core/__init__.py

```diff
@@ -108,86 +111,87 @@ def version_info() -> str:
     return _VERSION_INFO if _VERSION_INFO is not None else __version__
 
 
+pydoc.text.docother = fldoc.docother.__get__(pydoc.text, pydoc.TextDoc)
+
+
+# TODO: Warn when following legacy variables are used.
```

Comment:
A straightforward way is to make all of the following properties with deprecation warning.